Alignment/OfflineValidation/plugins/PixelBaryCentreAnalyzer.h

```diff
@@ -0,0 +1,117 @@
+#ifndef Alignment_OfflineValidation_PixelBaryCentreAnalyzer_H
```

Comment:
`cmssw` plugins should not have header files, but be fully contained in the .cc file.

Reply:
Done